cms/templates/index.html

```diff
@@ -318,8 +318,12 @@ <h3 class="course-title">${course_info['display_name']}</h3>
         <li class="archived-courses-tab"><a>${_("Archived Courses")}</a></li>
         % endif
         % if libraries_enabled:
+        % if use_new_library_editor:
+        <li><a href="${frontend_app_library_authoring_url}">${_("Libraries")}</a></li>
+        %else:
```

Comment:
I imagine that you'll also want to hide the `+ New Library` button on this page using similar logic.

Reply:
Probably, though I was unsure.  In any case, done.